oadr/oadr/manager/VENManager.cpp

```diff
@@ -338,11 +353,7 @@ void VENManager::run()
 
 void VENManager::stop()
 {
-	std::unique_lock<std::mutex> lock(m_mutex);
-
 	m_shutdown = true;
-
-	m_condition.notify_all();
```

Comment:
Is it ok to not call notify_call?

Reply:
Thanks for that comment. I missed the fact that it allows to notify waiting threads to finish quicker. That is important especially when the waiting time may long. I pushed a new commit, where I changed my approach to testing.